src/main/java/com/devcourse/java/calculator/constant/ExceptionConstant.java

```diff
@@ -0,0 +1,11 @@
+package com.devcourse.java.calculator.constant;
+
+public class ExceptionConstant {
+
```

Comment:
클래스명을 보고 내용을 대충 유추할 수 있게 네이밍해주세요!

이런 유틸성 클래스는 불변성을 가져야 해서 외부에서의 변경 가능성을 막아주는게 좋습니다.
final 키워드로 클래스 상속을 막고, private 생성자로 객체 생성을 막아주세요~

```suggestion
public final class ExceptionMessageConstant {
   private ExceptionMessageConstant() {
      // 여기서 예외 던져줘도 됩니다
   }
```

Reply:
내부 필드만 변경 못하게하면 된다는거를 생각하느라 외부에서의 클래스 불변성에 대해 놓친것 같습니다! 적용하겠습니다!!